src/protobuf-net.Test/Nullables/Abstractions/CollectionsWithNullsTestsBase.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ProtoBuf.Test.Nullables.Abstractions
+{
+    public abstract class CollectionsWithNullsTestsBase
+    {
+        private readonly ITestOutputHelper _log;
+
+        public CollectionsWithNullsTestsBase(ITestOutputHelper log)
+        {
+            _log = log;
+            Setup();
+        }
+
+        protected virtual void Setup() { }
+
+        protected T SerializeAndDeserialize<T>(T instance)
```

Comment:
probably not needed; unless you're using `ms` to check the payload, this is just `Serializer.DeepClone(instance)`, with extra steps

Reply:
removed Serialize&Deserialize and using DeepClone now